packages/jest-runtime/src/transform.js

```diff
@@ -25,7 +25,8 @@ const vm = require('vm');
 const VERSION = require('../package.json').version;
 
 type Options = {
-  isInternalModule: boolean,
+  isInternalModule?: boolean,
+  instrument?: boolean,
```

Comment:
we can pass either nothing or `{internalModule: true}` or `{instrument: false}` right now. So the whole thing is optional, as well as options inside